Core/Services/Auth/PasswordResetService.cs

```diff
@@ -11,15 +11,14 @@
 
 namespace Core.Services.Auth
 {
-    class PasswordResetService :  IPasswordResetService
+    class PasswordResetService : AuthServicesProvider, IPasswordResetService
     {
-        private readonly UserManager<User> _userManager;
-        private readonly IConfiguration _config;
-        private readonly IEmailService _emailService;
-        public PasswordResetService(UserManager<User> userManager, IConfiguration config, IEmailService emailService) {
-            _userManager = userManager;
-            _config = config;
-            _emailService = emailService;
+        private readonly IAdditionalAuthMetods _additionalAuthMetods;
+
+        public PasswordResetService(UserManager<User> userManager, IConfiguration config, IEmailService emailService, IAdditionalAuthMetods add) 
+            : base(userManager, config, emailService)
+        {
+            _additionalAuthMetods = add;
```

Comment:
Same as before, change 'add' to proper name

Reply:
Fixed